.changeset/thin-guests-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"fuels-wallet": minor
+---
+
+remove bech32
```

Comment:
This changeset is what goes to the Github Release description and we are aiming for more detailed releases, so I'd suggest we add more information here. 
For example:
- More detailed description of where bech32 address got removed (e.g. displayed wallet address, send/receive screens)
- Maybe the motivation behind this change